python/cudf/cudf/io/csv.py

```diff
@@ -106,6 +110,16 @@ def read_csv(
         index_col=index_col,
     )
 
+    if cudf.get_option("default_integer_bitwidth") == 32:
+        # Any 64-bit integer column unspecified in dtype is downcast to 32-bit.
+        df = _cast_integer_64bit_to_32bit(df, dtype)
```

Comment:
These utility functions seem highly specific. Can we generalize now so it becomes easier to support other default inferred types in the future?